src/routes.ts

```diff
@@ -3,10 +3,21 @@ import userControllers = require('./controllers/users');
 
 class routes{
 
-    constructor(app:Restify.Server){
+    constructor(server:Restify.Server){
+
+        server.post("/users/login/", userControllers.login);
+        server.get("/users/logout", userControllers.logout);
+        
+        /*********** Documentation routes ***********/
         
-        app.post("/users/login/", userControllers.login);
-        app.get("/users/logout", userControllers.logout);
+        // /docs does not render the css correctly, so redirect to /docs/
+        server.get('/docs', function(req, res, next){
+            res.header('Location', '/docs/');
+            res.send(302);
+        });
+
+        server.get(/\/docs\/?.*/, Restify.serveStatic({directory: './swagger',default: 'index.html'}));
```

Comment:
For now yeah. We can change it so that it only does this when the node environment is set to develop.  There are a couple other options.  We can either set up another server, or use Swagger and not Swagger-UI.  Swagger requires that you user their middle ware in all of your routes so that it has knowledge of all of route parameters, request types, response data, etc.